src/Plugin/ApigeeFieldStorageFormat/CSV.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License version 2 as published by the
+ * Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc., 51
+ * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+namespace Drupal\apigee_edge\Plugin\ApigeeFieldStorageFormat;
+
+use Drupal\apigee_edge\Plugin\ApigeeFieldStorageFormatInterface;
+
+/**
+ * @ApigeeFieldStorageFormat(
+ *   id = "csv",
+ *   label = "CSV",
+ *   fields = {
+ *     "boolean",
+ *     "float",
+ *     "integer",
+ *     "decimal",
+ *     "list_float",
+ *     "list_integer",
+ *     "list_string",
+ *     "string",
+ *     "string_long",
+ *   },
+ *   weight = -1,
+ * )
+ */
+class CSV implements ApigeeFieldStorageFormatInterface {
```

Comment:
Have you opened an issue in Symfony's issue queue regarding the CSVEncoder header issue? Please leave its link here.